tick/inference/base/learner_optim.py

```diff
@@ -52,21 +53,30 @@ class LearnerOptim(ABC, Base):
         Record history information when ``n_iter`` (iteration number) is
         a multiple of ``record_every``
 
+    random_state : int seed, RandomState instance, or None (default)
+        The seed that will be used by stochastic solvers. Used in 'sgd',
+        'svrg', and 'sdca' solvers
```

Comment:
It should be moved to Other Parameters after `elastic_net_ratio` (it should be in the same order as in the constructor)